kahuna/public/js/components/gr-image-metadata/gr-image-metadata.html

```diff
@@ -279,3 +279,40 @@
         </dd>
     </dl>
 </div>
+<div class="image-info">
```

Comment:
Is it possible/reasonable to componentise this control, so it can be shared with the search view (only handling single image case for now)?